test/perf/array/indexing.jl

```diff
@@ -168,7 +168,18 @@ function makearrays{T}(::Type{T}, sz)
     outersz = (sz[1]+1,sz[2]+2)
     B = reshape(convert(Vector{T}, [1:prod(outersz);]), outersz)
     Asub = sub(B, 1:sz[1], 2:sz[2]+1)
+    Asub2 = sub(A, :, :)
+    Asub3 = sub(AS, :, :)
+    C = reshape(1:4*prod(sz), 4, sz...)
+    Asub4 = slice(C, 1, :, :)
+    Asub5 = sub(C, 1, :, :)
+    Asub6 = slice(ArrayLS(C), 1, :, :)
+    Asub7 = sub(ArrayLS(C), 1, :, :)
+    Asub8 = slice(slice(C, :, :, :), :, 2:sz[2]+1)
+    Asub9 = slice(slice(ArrayLS(C), :, :, :), :, 2:sz[2]+1)
+
     Bit = trues(sz)
-    (A, AF, AS, ASS, Asub, Bit,)
+    # (A, AF, AS, ASS, Asub, Bit,)
+    (Asub, Asub2, Asub3, Asub4, Asub5)
```

Comment:
I've restored it back to the original for now.  It adds quite a bit of time and complexity to these tests… we may eventually want to create a SubArray perf suite since there are so many different performance-sensitive permutations.